osu.Game/Rulesets/UI/DrawableRuleset.cs

```diff
@@ -349,6 +349,9 @@ private void applyRulesetMods(IReadOnlyList<Mod> mods, OsuConfigManager config)
             foreach (var mod in mods.OfType<IApplicableToDrawableRuleset<TObject>>())
                 mod.ApplyToDrawableRuleset(this);
 
+            foreach (var mod in mods.OfType<ICreateReplay>())
+                SetReplayScore(mod.CreateReplayScore(Beatmap, mods));
```

Comment:
I don't understand. I thought the whole point of this PR was so that this wasn't a thing? Doesn't this result in creating it twice, once here and another time in `PlaySongSelect`?

Reply:
Overlooked this; should be fine to remove now.

Reply:
Actually, this was added for the editor. Didn't realise this was in `DrawableRuleset` itself. I guess it should be in `DrawableEditorRuleset`?

Reply:
I'd highly prefer it in `DrawableEditorRuleset`, yes.